controller.py

```diff
@@ -78,6 +84,10 @@ def annotate_feed(self, feed):
         feed.add_link_to_feed(
             feed.feed, href=search_url, rel="search", type=OPENSEARCH_MEDIA_TYPE
         )
+        register_url = self.app.url_for("register")
+        feed.add_link_to_feed(
+            feed.feed, href=register_url, rel="register"
+        )
```

Comment:
I assume we're using the semantics of the 'register' link relation defined in the Authentication for OPDS spec.